themes/gauge_theme/layout.html

```diff
@@ -27,7 +27,7 @@
 {%- endif %}
 
 {% set css_files = ['https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css'] %}
-{% set script_files = script_files + ['https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js', '_static/js/theme.js'] %}
+{% set script_files = script_files + ['https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js','//preview.gauge.org/assets/javascripts/site.js', '_static/js/theme.js'] %}
```

Comment:
Why do we need `site.js`'s reference here?

Reply:
It has script that performs action on navbar, and get Github star count. Zabil didnt wanted same thing running on both so I have referenced the script